lib/mayaUsd/python/wrapAdaptor.cpp

```diff
@@ -231,21 +328,41 @@ void wrapAdaptor()
               .def("RegisterTypedSchemaConversion", &::RegisterTypedSchemaConversion)
               .staticmethod("RegisterTypedSchemaConversion");
 
-    class_<UsdMayaSchemaAdaptor, UsdMayaSchemaAdaptorPtr>("SchemaAdaptor")
+    class_<UsdMayaAttributeAdaptor>("AttributeAdaptor")
         .def(!self)
+        .def("__repr__", _AttributeAdaptor__repr__)
+        .def("GetName", &UsdMayaAttributeAdaptor::GetName)
+        .def("Get", _AttributeAdaptor_Get)
+        .def("Set", _AttributeAdaptor_Set)
+        .def("GetAttributeDefinition", &UsdMayaAttributeAdaptor::GetAttributeDefinition);
+
+    class_<SchemaAdaptorWrapper, boost::noncopyable> c("SchemaAdaptor", boost::python::no_init);
```

Comment:
The schema adaptor needs to expose its virtual functions if we want the Python wrapper for schema API adaptor to be able to use it as base class in Python-land.